CMakeLists.txt

```diff
@@ -271,6 +318,19 @@ if (USE_PACKAGE_MANAGER)
             CONAN_PKG::libnoise
             CONAN_PKG::zlib
     )
+    if(BUILD_TEST)
```

Comment:
Same goes here. See #148 for reference, i've added an EXTRA_LIBS variable there.

Reply:
See reply above